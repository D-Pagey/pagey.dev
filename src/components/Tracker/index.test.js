import React from 'react';
import { Tracker } from '.';

const props = {
    details: [
        {
            id: '123',
            repo: { name: 'Gaming Page' },
            payload: {
                commits: [{ message: 'This is my commit message', sha: '11111' }]
            },
            created_at: '2020-07-02T19:11:46Z'
        }
    ],
    name: 'Andy'
};

describe('Tracker component', () => {
    it('should render', () => {
        const { container } = render(<Tracker {...props} />);
        expect(container.firstChild).toMatchSnapshot();
    });
});
