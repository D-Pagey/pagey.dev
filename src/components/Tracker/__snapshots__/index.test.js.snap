// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tracker component should render 1`] = `
.c0 {
  border: 1px solid black;
  border-radius: 5px;
  box-sizing: border-box;
  padding: 1rem 0;
  width: 100%;
}

.c1 {
  margin: 0;
  padding: 0.5rem 0 1rem;
  text-align: center;
}

.c2 {
  color: red;
}

<div
  class="c0"
>
  <h2
    class="c1"
  >
    Andy
    's Latest Commits
  </h2>
  <ul>
    <li>
      <p>
        Repo: 
        Gaming Page
      </p>
      <p>
        Commit:
         
        <a
          href="https://github.com/Gaming Page/commit/11111"
          rel="noopener noreferrer"
          target="_blank"
        >
          This is my commit message
        </a>
      </p>
      <p>
        Date:
         
        <span
          class="c2"
          color="red"
        >
          2nd July 2020 20:11
        </span>
      </p>
    </li>
  </ul>
</div>
`;
